packages/superset-ui-chart/test/components/SuperChart.test.tsx

```diff
@@ -33,10 +37,18 @@ describe('SuperChart', () => {
     });
   });
 
+  beforeEach(() => {
+    restoreConsole = mockConsole();
```

Comment:
are these all added because nimbus would make warning throw? might be worth making comments for that since it's not obvious.

Reply:
comments added